charts/provider-postgresql/README.gotmpl.md

```diff
@@ -13,6 +13,33 @@ Edit the README.gotmpl.md template instead.
 * Always upgrade the CRDs before upgrading the Helm release.
 * Watch out for breaking changes in the {{ title .Name }} release notes.
 
+## Webhook support
+
+This chart is capable of rendering the `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration` required for the operator.
+While you can disable webhook support with `webhook.enabled=false`, you'll lose important business functionality.
+But it may be easier to set up in testing environments.
+
+In order for webhooks to work, Kubernetes requires a Secret with TLS.
+The webhook configuration spec requires the CA bundle to be set as well so that Kubernetes trusts the certificate that is mounted in the operator.
+
+You can set `webhook.existingSecretName` to a Secret that is managed outside of this chart.
+
+### Automatic Certificate Provisioning on OpenShift 4
+
+On Openshift 4, you can make use of the automatic service serving certificate provisioning as documented here:
```

Comment:
Maybe add a note, that OpenShift creates the secret, otherwise the naming `existingSecretName` could be somewhat confusing.

Reply:
The chart itself doesn't make assumptions where the secrets are coming from resp. how they are created.
What exactly is confusing? Do you have a better name?

Reply:
It's not quite clear if the OpenShift Certificate provisioner creates a new secret, or if that also has to be an existing one.

Reply:
If you read the Openshift 4 documentation linked below, you will know that the cert provisioner will create the secret :) so that's a case of RTFM? :P 

We could consider renaming to `webhook.externalSecretName` if that's better.